webpack.config.js

```diff
@@ -53,6 +54,10 @@ module.exports = [
         }
       ])
     ],
+    optimization: {
+      minimize: true,
+      minimizer: [new TerserPlugin()]
+    },
```

Comment:
No need to add it here, as it(webpack job) is not really bundling anything (null-loader), just used to copy assets.